internal/pproflogging/pproflogging.go

```diff
@@ -127,14 +164,19 @@ func parseProfileConfigs(bufSizeB int, ppconfigs string) map[ProfileName]*Profil
 		flagValue := ""
 
 		if len(profileFlagNameValuePairs) > 1 {
+			// only <key>=<value? allowed
 			flagValue = profileFlagNameValuePairs[1]
 		}
 
-		flagKey := ProfileName(strings.ToLower(profileFlagNameValuePairs[0]))
+		flagKey := ProfileName(profileFlagNameValuePairs[0])
+		if flagKey == "" {
```

Comment:
is it required to strip the value before checking for empty ?

Reply:
The values should be stripped before they reach this point.  Regardless, I can add a  unit test later to ensure they are stripped.